kommunicate/src/main/java/io/kommunicate/Kommunicate.java

```diff
@@ -513,4 +518,8 @@ private static String generateUserId() {
         }
         return text.toString();
     }
+
+    public static void getAwayMessage(Context context, Integer groupId, KmAwayMessageHandler handler) {
```

Comment:
This method is not returning anything but naming with get makes it looks like it is going to return something. Also static method running something in async, is that correct way?

Reply:
That method is will be returning the message in the Callback KmAwayMessageHandler.
It is just a wrapper that executes an async task. It is good practice as we do not have to implement the async task everywhere, just calling the method works. If android stops support for async task, then we can directly change the logic inside the method and the usages won't be effected.

Reply:
Naming convention in above example is not 'get', its 'on'. All get in above example are returning object

Reply:
Done. Changed it to loadAwayMessage